src/app/pages/TransactionDetailPage/index.tsx

```diff
@@ -341,6 +372,54 @@ export const TransactionDetailView: FC<{
 
           <dt>{t('common.gasLimit')}</dt>
           <dd>{transaction.gas_limit.toLocaleString()}</dd>
+
+          {transaction.encryption_envelope && (
+            <>
+              <dt>{t('transactions.encryption.format')}</dt>
+              <dd>
+                {transaction.encryption_envelope.format} &nbsp; <TransactionEncrypted />
+              </dd>
+
+              <dt>{t('transactions.encryption.publicKey')}</dt>
+              <dd>
+                <Typography variant={'mono'} sx={{ overflowWrap: 'anywhere' }}>
+                  {transaction.encryption_envelope.public_key}
+                </Typography>
+              </dd>
+
+              <dt>{t('transactions.encryption.dataNonce')}</dt>
+              <dd>
+                <Typography variant={'mono'} sx={{ overflowWrap: 'anywhere' }}>
+                  {transaction.encryption_envelope.data_nonce}
+                </Typography>
+              </dd>
+
+              {transaction.encryption_envelope.data !== undefined && (
+                <>
+                  <dt>{t('transactions.encryption.encryptedData')}</dt>
+                  <dd>
+                    <ShowHideData data={transaction.encryption_envelope.data} threshold={300} />
+                  </dd>
+                </>
+              )}
+
+              <dt>{t('transactions.encryption.resultNonce')}</dt>
+              <dd>
+                <Typography variant={'mono'} sx={{ overflowWrap: 'anywhere' }}>
+                  {transaction.encryption_envelope.result_nonce}
```

Comment:
font-weight: 400

Reply:
OK I added font-weight to all the new mono typographies .